### PR TITLE
Update cats, cats-effect and http4s versions

### PR DESCRIPTION
**Pull request by janschultecom**
Solves #3

### CODE REVIEW

File: project/versions.scala [hunk 1]
```diff
@@ -16,13 +16,13 @@
 //: ----------------------------------------------------------------------------
 
 object V {
-  val http4sArgonaut61 = "0.16.0"
-  val http4s           = "0.18.9"
+  val http4sArgonaut61 = "0.20.0-M4"
```
Comment (rossabaker) [inline project/versions.scala]: Is this used anymore?  We haven't published http4s-argonaut61 for a long time.

**Review by rossabaker — COMMENTED** [review]
Thanks.  Generally looks correct to me.  One possible bit of cruft.

File: project/versions.scala (hunk 1 above)
Comment (janschultecom) [reply project/versions.scala]: Done

**Review by rossabaker — APPROVED** [review]

**Comment by janschultecom** [comment]
Can we merge this?

**Comment by rossabaker** [comment]
I don't have release keys anymore.  I would ask on Nelson Gitter about a release if you need one.

**Pull request merged**